app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/LimitPickerDialog.java

```diff
@@ -0,0 +1,106 @@
+package de.tum.in.tumcampusapp.component.tumui.calendar;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.view.View;
+import android.widget.Button;
+import android.widget.NumberPicker;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.utils.Const;
+import de.tum.in.tumcampusapp.utils.Utils;
+
+
+
+
+
+class LimitPickerDialog extends Dialog {
+    private Button cancelButton;
+    private Button saveButton;
+    private NumberPicker numberPickerMin;
+    private NumberPicker numberPickerMax;
+    protected Context context;
+    private List<LimitPickerDialogListener> listeners = new ArrayList<LimitPickerDialogListener>();
```

Comment:
I think, we can leave out the generic type on the right side.
Since JDK 7 it should be sufficient to write:
private List\<LimitPickerDialogListener\> listeners = new ArrayList<>()